.semaphore/semaphore.yml

```diff
@@ -151,6 +155,81 @@ blocks:
           commands:
             - bats tests/sem_version_focal/${TEST}.bats
 
+  - name: Sem Version Tests Jammy ARM
+    dependencies:
+      - "Build local CLIs"
+    task:
+      agent:
+        machine:
+          type: r1-standard-4
+          os_image: ubuntu2204
+      prologue:
+        commands:
+          - wget https://github.com/semaphoreci/artifact/releases/download/v0.6.2/artifact_Linux_arm64.tar.gz -O /tmp/artifact_Linux_arm64.tar.gz
+          - sudo tar -xf /tmp/artifact_Linux_arm64.tar.gz -C /usr/local/bin/
+          - checkout
+          - artifact pull workflow bin/linux/amd64/cache -d cache-cli/bin/linux/amd64/cache
+          - artifact pull workflow bin/linux/arm64/cache -d cache-cli/bin/linux/arm64/cache
+          - artifact pull workflow bin/darwin/amd64/cache -d cache-cli/bin/darwin/amd64/cache
+          - artifact pull workflow bin/linux/amd64/sem-context -d sem-context/bin/linux/amd64/sem-context
+          - artifact pull workflow bin/linux/arm64/sem-context -d sem-context/bin/linux/arm64/sem-context
+          - artifact pull workflow bin/darwin/amd64/sem-context -d sem-context/bin/darwin/amd64/sem-context
+          - bash release/create.sh
+          - source release/install_in_tests.sh
+          - git submodule init && git submodule update
+          - sudo ./tests/support/bats-core/install.sh /usr/local
+      jobs:
+        - name: Sem Version
+          matrix:
+            - env_var: TEST
+              values:
+                - elixir
+                - erlang
+                - gcc
+                - go
+                - java
+                - kubectl
+                - node
+                - php
+                - python
+                - ruby
+                - scala
+          commands:
+            - bats tests/sem_version_jammy_arm/${TEST}.bats
```

Comment:
Why do we need separate `tests/sem-version_jammy_arm` tests? Can't we use the same sem-version tests we already have?